run_uh.py

```diff
@@ -121,22 +121,18 @@ def excepthook(exception_type, value, tb):
 		with open(outfilename, 'a') as f:
 			traceback.print_exception(exception_type, value, tb, file=f)
 		traceback.print_exception(exception_type, value, tb)
-		print('')
-		print(T('Unknown Horizons has crashed.'))
-		print('')
+		print('\n', T('Unknown Horizons has crashed.', '\n'))
```

Comment:
Either way is probably fine.

However, talking about newlines beeing obvious, your change introduced an error.

`print('\n', T('Unknown Horizons has crashed.', '\n'))`

should be

`print('\n', T('Unknown Horizons has crashed.'), '\n')`

Notice the placement of the closing paranthesis.